src/components/search/Search.jsx

```diff
@@ -32,7 +33,17 @@ const Search = () => {
   });
 
   const config = getConfig();
-
+  const courseIndex = useMemo(
+    () => {
+      const client = algoliasearch(
+        config.ALGOLIA_APP_ID,
+        config.ALGOLIA_SEARCH_API_KEY,
+      );
+      const cIndex = client.initIndex(config.ALGOLIA_INDEX_NAME);
+      return cIndex;
+    },
+    [], // only initialized once
+  );
```

Comment:
Is it necessary to initialize a separate index for typeahead search? can't we use one that is already initialized?